### DISCUSSION TITLE
missing record option on home screen following update

### DISCUSSION THREAD

**Discussion by chrisaww** [osmandapp/OsmAnd#12886] [Q&A]
OsmAnd has just been updated on my Galaxy phone. Seems to start up and show current location but the record button associated with the record tack plugin is missing ! Also missing is all of the data that used to be presented in the top right hand corner. I have check ed plugin settings and it is showing that it is active.
thanks for any advice

**Comment by pebogufi** [comment]
There was a bug. see report in
https://github.com/osmandapp/OsmAnd/issues/12828
There you will find a workaround until you update. 
(Open plugins, disable + enable Trip Recording plugin)
